SDWebImage/Core/SDAnimatedImageView.h

```diff
@@ -52,6 +53,10 @@
  `< 0.0` is not supported currently and stop animation. (may support reverse playback in the future)
  */
 @property (nonatomic, assign) double playbackRate;
+
+/// Asynchronous setup animation playback mode. Default mode is SDAnimatedImagePlaybackModeNormal.
+@property (nonatomic, assign) SDAnimatedImagePlaybackMode playbackMode;
+
```

Comment:
We can expose the `SDAnimatedImagePlayer` this property into the open API of SDAnimatedImageView. With some code comment such as `Setting directly to the player will cause some animation status not get synchronized. This property is used for some advanced usage or debugging usage.`

Reply:
If expose the `SDAnimatedImagePlayer ` ,`SDAnimatedImageView`'s some property like` playbackRate` and `maxBufferSize` should be hidden？

Reply:
Just keep both. This is stable API from 5.0...

Reply:
In the future maybe we can remove the same API exposed to the ImageView && Player.